### DISCUSSION TITLE
Adding browser/os/countries logo/flag?

### DISCUSSION THREAD

**Discussion by ErisFY** [plausible/analytics#957] [Anything Else]
This might come across as negligible, but are there any plans to add logos next to the OS and Browser in the Device section? As well as add a list of countries underneath the map, and have flags next to each country/city? I know that the country list is visible by clicking on Detail. 

It's just an aesthetic feature, but I feel like it'd make the whole dash more dynamic looking and allow users to take info in at a glance.

**Comment by metmarkosaric** [comment]
logos/icons for OS and browser could be nice so it fits with icons in devices and favicons in top sources

**Comment by fyrk** [comment]
I created a Stylus theme that does exactly this -> https://github.com/FlorianRaediker/plausible-more-logos
It adds logos for browser and OS, though no flags for countries.
